EPRN.UnitTests/Portal/Services/ExporterServiceTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using EPRN.Portal.Resources;
+using EPRN.Portal.Services;
+using EPRN.Portal.Services.HomeServices;
+using EPRN.Portal.ViewModels;
+
+namespace EPRN.UnitTests.Portal.Services
+{
+    [TestClass]
+    public class ExporterServiceTests
+    {
+        private ExporterHomeService _exporterHomeService;
+
+        [TestInitialize]
+        public void Init()
+        {
+            _exporterHomeService = new ExporterHomeService();
+        }
+
+        [TestMethod]
+        public async Task GetHomePage_ReturnsValidModel()
+        {
+            // Arrange
+
+            // Act
+            var viewModel = await _exporterHomeService.GetHomePage();
+
+            // Assert
+            Assert.IsNotNull(viewModel);
+            Assert.AreEqual(typeof(HomepageViewModel), viewModel.GetType());
+        }
+
+        [TestMethod]
+        public async Task GetHomePage_Returns_Correct_Cards()
+        {
+            // Arrange
+
+            // Act
+            var viewModel = await _exporterHomeService.GetHomePage();
+
+            // Assert
+            Assert.IsNotNull(viewModel);
+            Assert.AreEqual(typeof(HomepageViewModel), viewModel.GetType());
+
+            Assert.IsTrue(viewModel.CardViewModels.Count() == 5);
+            Assert.IsTrue(viewModel.CardViewModels[0].Title == HomePageResources.HomePage_Waste_Title);
```

Comment:
Let's try to test for another tile just to cover a little more base and switch up the tests from the other service tests. The user role specific one is fine

Reply:
Done.